Packages/com.unity.inputsystem/Documentation~/Debugging.md

```diff
@@ -28,41 +23,45 @@ The debugger shows a tree breakdown of the state of the input system.
 |Users|Only visible when there is one or more `InputUser` instances. See [documentation](UserManagement.md).<br><br>Lists all currently active uers along with their active control schemes and devices, all their associated actions as well as the controls they are bound to.<br><br>Note that `PlayerInput` uses `InputUser` underneath. This means that when using `PlayerInput` components, each player will have an entry here.<br><br>See [Debugging Users/PlayerInput](#debugging-usersplayerinput).|
-|Settings||
-|Metrics||
+|Settings|Shows the currently active Input System [settings](Settings.md).|
+|Metrics|Shows some statics about Input System resource usage.|
 
-### Debugging Remotely
+### Debugging Devices
 
-You can connect input debugger to a player running on a device. This makes input activity from the player observable in the editor. The mechanism used by this is Unity's `PlayerConnection`, i.e. the same mechanism by which the Unity profiler can be connected to a player.
+You can double-click on any [input device](Devices.md) in the `Devices` list in the debugger window to open a window showing information about the device, including real-time state information for its controls.
 
->NOTE: At the moment, debugging input in players is restricted to seeing devices and events from connected players. There is no support yet for seeing other input-related data from players.
+IMAGE
 
-    ////TODO: explain "Remote Devices"
+The top of the device window shows some general information about the specific device (such as name, manufacturer, serial number, etc).
 
-### Debugging Devices
+Below that, you you see a view listing the devices controls, and their state. This is very useful when debugging input issues, as it allows you to verify if the data the input system is receiving from the input device is what you'd expect it to be in the first place. At the top of this view you will see the following buttons:
+
+* `State`: Show the current state of the device in a new window. This is identical to the information as shown in this view, but is not updated, so you can take a slice of input state data and take the time to inspect it as needed.
+
+* `HID Descriptor`: Only shown for devices connected using the HID protocol. This will open a window showing the detailed HID specifications for the device and each of it's logical controls.
 
->NOTE: For an alternative way to debug/visualize controls and/or devices, see [Visualizing Controls](#visualizing-controls).
+Finally, in the bottom of the window, you can inspect the event stream coming from the device. Any [Input Event](Events.md) generated by the device will be shown here. You can double-click any event in the list to inspect the full device state at the time of the event.
```

Comment:
There's also the option to select two (IIRC actually any number >=2) events and right-click and select "Compare" to see only differences between the state payloads.

Reply:
Oh, did not know about that. Will add it.